README.md

```diff
@@ -1,34 +1,89 @@
-# RpaEmAssemblyWebapp
+# @hmcts/dg-docassembly-webcomponent 
+[![Coverage Status](https://coveralls.io/repos/github/hmcts/dg-docassembly-webcomponent/badge.svg?branch=master)](https://coveralls.io/github/hmcts/dg-docassembly-webcomponent?branch=upload-npm-in-pipeline)
+[![Build Status](https://travis-ci.com/hmcts/dg-docassembly-webcomponent.svg?branch=master)](https://travis-ci.com/hmcts/dg-docassembly-webcomponent)
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 7.1.4.
+The docassembly webcomponent is an angular library that provides components to generate a document based on a predefined template customised with user input.
+The library is made up of 3 main components:
+- assembly-viewer, which is the parent component that acts as a wrapper that wires the other components together
+- form-viwer, which retrieves and displays the input form that allows the user to cutomise the generated document
```

Comment:
viewer*

Reply:
customise*

Reply:
might be abit picky but its not customising a generated document, but customise the document which can be generated with the updated info.